content/issues/63.markdown

```diff
@@ -0,0 +1,32 @@
+---
+title: Issue 63
+published: 2017-07-13
+---
+
+Welcome to another issue of Haskell Weekly!
+Haskell is a purely functional programming language that focuses on robustness, concision, and correctness.
+This is a weekly summary of what's going on in its community.
+
+## Featured
+
+todo
+
+## Package of the week
+
+todo
+
+## Call for Participation
+
+Are you interested in contributing to open-source Haskell projects but not sure where to start?
+Here are some tasks from the Haskell community for you to pick and get started!
+
+* [stack: "stack new" can't be pointed at an intranet site](https://github.com/commercialhaskell/stack/issues/2804)
+* [aeson: Hindent?](https://github.com/bos/aeson/issues/527)
+* [miso: phantomjs2 fails to build on OSX](https://github.com/dmjio/miso/issues/160)
+* [esqueleto: Make examples for the README buildable](https://github.com/bitemyapp/esqueleto/issues/4)
+* [cabal: Print out more information about the effective configuration when you build](https://github.com/haskell/cabal/issues/3945)
+* [text: Add tshow function](https://github.com/bos/text/issues/183)
```

Comment:
This is super minor, but I prefer using hyphens (`-`) for lists instead of asterisks (`*`).